frontend/js/components/blocks/BlockEditorItem.vue

```diff
@@ -71,7 +71,7 @@
     },
     data () {
       return {
-        visible: true,
+        visible: false,
```

Comment:
If less then 4 expandables, we can show them

Reply:
mounted () {
      // if there are blocks, these should be all collapse by default
      this.$nextTick(function () {
        if (this.blocks(this.editorName) && this.blocks(this.editorName).length > 3) this.collapseAllBlocks()
      })
    }

in blocks.vue